acoular/spectra.py

```diff
@@ -179,12 +179,7 @@ class PowerSpectra(BaseSpectra):
 
     #: The :class:`~acoular.calib.Calib` object that provides the calibration data,
     #: defaults to no calibration, i.e. the raw time data is used.
-    #: **deprecated, will be removed in version 25.01**: use
-    #: :attr:`~acoular.sources.TimeSamples.calib` property of :class:`~acoular.sources.TimeSamples`
-    #: objects
-    calib = Property(desc='calibration object (deprecated, will be removed in version 25.01)')
-
-    _calib = Instance(Calib)
+    calib = Instance(Calib)
```

Comment:
Please remove the calib capabilities from the PowerSpectra class (you can parse for calib):
* remove the calib attribute
* remove the _handle_dual_calibration, _get_calib and _set_calib functions
* remove calib attributes from all sublcasses (including PowerSpectraImport)